events/janus_mqttevh.c

```diff
@@ -142,6 +145,7 @@ typedef struct janus_mqttevh_context {
 
 	/* Connection data - authentication and url */
 	struct {
+        int mqtt_version;
```

Comment:
Indentation is broken: please use tabs and not spaces (not sure if the same happens in other parts of the code, in case please fix it there too).